zulipterminal/ui_tools/views.py

```diff
@@ -792,6 +792,12 @@ def __init__(self, controller: Any) -> None:
     def keypress(self, size: urwid_Size, key: str) -> str:
         if is_command_key('GO_BACK', key) or is_command_key('HELP', key):
             self.controller.exit_popup()
+        elif is_command_key('GO_UP', key) \
+                or is_command_key('SCROLL_UP', key):
+            key = 'up'
+        elif is_command_key('GO_DOWN', key) \
+                or is_command_key('SCROLL_DOWN', key):
+            key = 'down'
```

Comment:
I would think we'd want the `SCROLL_*` keys (and `GO_TO_BOTTOM`) to do what they would normally do, not just move up/down.